addons/mail/__init__.py

```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
 
 from . import models
+from . import tests
```

Comment:
why is needed this? 🤔 usually, `tests` folder is not put in the module init

Reply:
Actually it is some magic that import tests if I am not wrong. Doing it explicitly does no harm.